test/routes/openConditions.test.js

```diff
@@ -276,3 +281,33 @@ describe('open conditions', () => {
       .expect('Location', `/tasklistRecat/12345`)
   })
 })
+
+describe('Validating Previous Sentences page', () => {
```

Comment:
Would this be more suitable in fieldValidation.test.js?

Reply:
Put the validation tests in /config/openConditions.test.js as the validation is in /config/openConditions.js